resources/views/tools/bread/edit-add.blade.php

```diff
@@ -2,6 +2,10 @@
 
 @if (isset($dataType->id))
     @section('page_title', __('voyager::bread.edit_bread_for_table', ['table' => $dataType->name]))
+    @php
+        $display_name = $dataType->display_name_singular;
```

Comment:
Not sure if a mistake or intentional, but shouldn't this be wrapped in a call to `getTranslatedAttribute()` as well?

Reply:
I remember in one place I left it because I thought it made more sense to leave it untranslated but I'll have to check it tomorrow to see what was the reason and if it still make sense.

Reply:
I checked again and the only place where $display_name is actually used in that file is in the breadcrumb on top because the other two places are overridden by the i18n interaction.
Anyway I remembered why I left it out but the reasoning doesn't look that strong anymore, so I think I can change that too.
Thanks for the review.